CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ x.y.z Release notes (yyyy-MM-dd)
 Upgraded realm-core from ? to ?
 Upgraded realm-sync from ? to ?
 
+4.3.2 Release notes (2020-02-04)
```

Comment:
The changelog entry should go in the above section, not add a new one. The flow for the changelog is that each PR adds entries to the x.y.z section for the next release, and then as part of the release process that is replaced with the actual version number and release date. After the release is complete, a new x.y.z section is added to the top of the changelog.